client/fingerprint/network_test.go

```diff
@@ -486,3 +486,98 @@ func TestNetworkFingerPrint_MultipleAliases(t *testing.T) {
 	sort.Strings(aliases)
 	require.Equal(t, expected, aliases, "host networks should match aliases")
 }
+
+func TestNetworkFingerPrint_HostNetorkReservedPorts(t *testing.T) {
```

Comment:
```suggestion
func TestNetworkFingerPrint_HostNetworkReservedPorts(t *testing.T) {
```

Reply:
Several times Go wouldn't find any matching test to run. Now I know why 🤦 

Thanks!